src/main/java/Eln.java

```diff
@@ -0,0 +1,105 @@
+
+import java.util.Scanner;
+
+
+public class Eln {
+    private static final String LINE = "____________________________________________________________";
+    private static Task[] tasksList = new Task[100]; // max number of tasks set to 100
```

Comment:
Plural form should be used on names representing a collection of objects! Also, a suggestion: can put the number 100 a a named constant :]